packit/config/common_package_config.py

```diff
@@ -220,6 +223,7 @@ def __init__(
         upstream_tag_exclude: str = "",
         upload_sources: bool = True,
         pkg_tool: Optional[str] = None,
+        version_update_mask: str = r"\d+\.\d+\.",
```

Comment:
As discussed, we can go with an empty string for now so we don't change anything for the existing users.